src/main/java/com/apieceofcoding/lecture11/Car.java

```diff
@@ -20,6 +20,10 @@ public void turnOn() {
 
     @Override
     public void fillUp(int fuel) {
+        if (fuel < 0) {
+            System.out.println("연료는 0 이상이어야 합니다.");
+            return;
```

Comment:
return 대신 예외처리하는 것이 어떨까요?
```suggestion
    throw new IllegalArgumentExeception();
```

Reply:
네 반영했습니다 😀